### PR TITLE
Change go-elektra module name to go.libelektra.org

### PR DESCRIPTION
**Pull request by tmakar**
## Basics

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which contains `_(my name)_`)
      **Please always add them to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - add a line in `doc/news/_preparation_next_release.md`
  - reformat the code with `scripts/dev/reformat-all`
  - make all unit tests pass
  - fix all memleaks
  - fix the CI itself (or rebase if already fixed)
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation (see [Documentation Guidelines](https://www.libelektra.org/devgettingstarted/documentation))
- [ ] I fixed all affected decisions (see [Decision Process](https://www.libelektra.org/decisions/decision-process))
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://www.libelektra.org/devgettingstarted/coding))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if everything is done** and no further pushes are planned by you.

### CODE REVIEW

**Comment by tmakar** [comment]
@atmaxinger @markus2330 this PR is also required for the next release!

File: doc/news/_preparation_next_release.md
Comment (atmaxinger) [inline doc/news/_preparation_next_release.md:291]: Is it really a _rename_ or is it just a change in URL?

File: src/bindings/go-elektra/kdb/benchmark_test.go
Comment (tmakar) [reply src/bindings/go-elektra/kdb/benchmark_test.go:7]: Yes, @markus2330 has changed it to the new url.

Comment (tmakar) [reply src/bindings/go-elektra/kdb/benchmark_test.go:7]: We will see if it is available in Go packages after we have merged on master.

File: src/bindings/go-elektra/.idea/workspace.xml
Comment (tmakar) [reply src/bindings/go-elektra/.idea/workspace.xml:1]: Oh, should not be checked in.

**Comment by tmakar** [comment]
TODO:

- Adapt the `README`for both to use new url

**Review by atmaxinger — APPROVED** [review]
LGTM

**Comment by tmakar** [comment]
@markus2330 I now checked and the problem we still have is that `go.libelektra.org` can not be used as it is already published to go packages. There is no way to remove an already published version from go packages. So, I would suggest using `goelektra.libelektra.org`. We then need to also change the metakey `go-import` in html accordingly.

**Comment by tmakar** [comment]
@atmaxinger you can merge after build passes!

**Comment by atmaxinger** [comment]
What's the point of merging if it doesn't work?

**Comment by tmakar** [comment]
> What's the point of merging if it doesn't work?

It might work, could override the existing module. I would give it a try. Without merging we can not know. Or we immediately change the url to `goelektra.libelektra.org`. @markus2330

**Comment by tmakar** [comment]
jenkins build libelektra please

**Comment by tmakar** [comment]
jenkins build libelektra please

**Comment by tmakar** [comment]
This PR is on hold until we have configured a redirect in metakey `go-import` to match `goelektra.libelektra.org`. This PR introduces the changed module name for the go binding (`goelektra.libelektra.org`).

**Review by markus2330 — APPROVED** [review]

**Pull request merged**
